bqskit/ir/gates/constant/ecr.py

```diff
@@ -0,0 +1,26 @@
+"""This module implements the ECRGate."""
+from bqskit.ir.gates.qubitgate import QubitGate
+from bqskit.ir.gates.constantgate import ConstantGate
+from bqskit.qis.unitary.unitarymatrix import UnitaryMatrix
+import math
+
+class ECRGate(ConstantGate, QubitGate):
```

Comment:
Please add some documentation that references a good resource for this gate and includes the matrix representation.